tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "outDir": "dist",
     "sourceMap": true,
     "strictNullChecks": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "types": ["@emotion/react/types/css-prop", "node", "jest"]
```

Comment:
If you're using Emotion's JSX namespace, you might want to consider using [React's automatic runtime](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) too. This means you no longer need to write `import React from `react` at the top of every module. It also unlocks certain performance improvements. Check out Source's [recommended `tsconfig` setup](https://theguardian.design/2a1e5182b/p/24c1c7-using-emotion/t/001a69)

Reply:
Thanks @SiAdcock, good shout! We'll definitely do that, possibly in a follow up PR if that sounds reasonable? (There are a few moving parts going on with upgrading Emotion/source across DCR and other tools so am hoping not to increase the scope of this PR further).